x-pack/plugins/viz_editor/public/pseudo_plugins/pie_chart_plugin/pie_chart_plugin.tsx

```diff
@@ -144,6 +155,9 @@ function getSuggestionsForField(
   field: DatasourceField,
   visModel: PieChartVisModel
 ): Suggestion[] {
+  if (!field.aggregatable) {
+    return [];
+  }
```

Comment:
In theory we can support raw values in the pie chart, right?

Reply:
Yes, but the current query building logic doesn't as it's always suggesting a count operation - this is why I left it out for now. We can and should allow this when the pie suggestions are improved